### PR TITLE
Generate Cargo.toml of message crate with an EmPy template, not CMake

### LINKED ISSUE #114: Generate Cargo.toml of message crate with an EmPy template, not CMake
Currently, the source files are generated with an EmPy template, but Cargo.toml is created from a CMake template. It would be good to create Cargo.toml from an EmPy template as well, for simplicity.

### PR DESCRIPTION
**Pull request by nnmm**
Closes #114

### CODE REVIEW

File: rosidl_generator_rs/resource/Cargo.toml.em [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[package]
+name = "@(package_name)"
+version = "0.2.0"
```
Comment (esteve) [inline rosidl_generator_rs/resource/Cargo.toml.em:3]: Can you add an EmPy comment with a reference to https://github.com/ros2-rust/ros2_rust/issues/116 ?

**Review by esteve — CHANGES_REQUESTED** [review]

File: rosidl_generator_rs/resource/Cargo.toml.em (hunk 1 above)
Comment (nnmm) [reply rosidl_generator_rs/resource/Cargo.toml.em:5]: Good idea, done.

Comment (esteve) [reply rosidl_generator_rs/resource/Cargo.toml.em:5]: Thanks.

**Review by esteve — APPROVED** [review]
Really nice, the less CMake magic we use, the better.

**Pull request merged**